lms/djangoapps/discussion/rest_api/utils.py

```diff
@@ -448,3 +451,14 @@ def get_course_id_from_thread_id(thread_id: str) -> str:
         'mark_as_read': False
     })
     return thread["course_id"]
+
+
+def is_only_student(course_key, user) -> bool:
```

Comment:
Hi, I have reviewed the PR The only thing missing here is a unit test for is_only_student. This function is mocked in all tests. Test can be added, something like this 
Add users with different roles and run the function against them to check its behavior without any mocking.